app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -1,3 +1,4 @@
+- display_grading_results = @submission.published? || can?(:grade, @submission)
```

Comment:
Thought it might be clearer as that will explain what `mark` is, and how `graded` differs from `published`. This way works as well unless we need it in multiple places 👍